csharp/src/Ice/IceSSL/SSLEngine.cs

```diff
@@ -137,52 +123,20 @@ internal void initialize()
 
             try
             {
-                using FileStream fs = File.OpenRead(certAuthFile);
-                byte[] data = new byte[fs.Length];
-                fs.Read(data, 0, data.Length);
-
-                string strbuf = "";
+                // Try using PEM format
                 try
```

Comment:
Maybe we should just look at the file extensions? I don't think they're ambitious.

Reply:
so use `LoadFromPemFile` for `.pem` insensitive compare, and everything else for `Import` which handles pfx/der?

Note `LoadFromPemFile` is the only way to get multiple certificates imported.

Reply:
If that makes sense to you too

Reply:
Reworked this code to check file extensions.